lib/graphql/adapters/delivery_adapter.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+module GraphQL
+  module Adapters
+    class DeliveryAdapter
+      attr_reader :executor_object, :executor_method
+
+      def initialize(executor_object:)
+        @executor_object = executor_object
+        @executor_method = executor_object.class::EXECUTOR_METHOD_NAME
+      end
+
+      def trigger(...)
```

Comment:
I use here "...", but It requires "ruby-2.7".

I checked ci-cd, and I saw minimal ruby versions - 2.7, but if we look at the documentation, I see the requirement for ruby-version 2.6

Reply:
@palkan please tell me, do we need to consider ruby 2.6 and avoid using "..." ?
If yes, I will fix it